mkdocs-base.yml

```diff
@@ -18,6 +18,8 @@ theme:
   logo: _static/percona-logo.svg
   favicon: _static/percona_favicon.ico
```

Comment:
wrong path. Change to _static/percona-favicon.ico

Reply:
done

Reply:
Done